### PR TITLE
fix: move max-width: 100px from menu triggers to icon triggers

### PR DESCRIPTION
**Pull request by chrisdo1**
### What has changed and why

Removed max-width: 100px that was causing the accordion trigger text to wrap between 1024px and 1030px. This max-width property was causing triggers inside the menu tray accordions to wrap between ~1000px and ~1024px.

#### Browser requirements

Your component should work in all of the following environments:

- [x] Latest 2 versions of Edge
- [x] Internet Explorer 11 (should be useable, not pixel perfect)
- [x] Latest 2 versions of Firefox (one on Mac OS, one of Windows OS)
- [ ] Firefox 60.7.2 (or latest version for Red Hat Enterprise Linux distribution)
- [x] Latest 2 versions of Chrome (one on Mac OS, one of Windows OS)
- [x] Latest 2 versions of Safari
- [x] Galaxy S9 Firefox
- [x] iPhone X Safari
- [x] iPad Pro Safari
- [x] Pixel 3 Chrome

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable to your PR.

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- <s>Tests have been updated to cover these changes.</s>
- [x] Browser testing passed.
- <s>Documentation (README.md, WHY.md, etc.) updated or added.</s>
- [ ] Link to the demo recording: []()
- [ ] Approved by designer.

### CODE REVIEW

File: elements/pfe-navigation/src/pfe-navigation-item.scss [hunk 1]
```diff
@@ -270,6 +270,7 @@ $variables: map-merge($variables, $nav-item-variables);
 :host([pfe-icon]) {
```
Comment (castastrophe) [inline elements/pfe-navigation/src/pfe-navigation-item.scss:270]: I might suggest combining this code with the ::slotted selector above for clarity:

```
::slotted([slot="trigger"]) {
  @extend %trigger;
  :host([pfe-icon]) & {
    margin: 0.25em 0 0 0 !important;
    max-width: 100px;
  }
}
```

**Comment by castastrophe** [comment]
@chrisdo1 Any test cases to be added or updated?  Needs a changelog but otherwise it look good :+1:

**Comment by castastrophe** [comment]
Demo page: https://5eb04a49bf41c70006abe058--happy-galileo-ea79c4.netlify.app/elements/pfe-navigation/demo

**Comment by chrisdo1** [comment]
@castastrophe I merged in the latest from master and updated the changelog. Can you give this an LGTM when you have a free minute? Thanks!

**Pull request merged**